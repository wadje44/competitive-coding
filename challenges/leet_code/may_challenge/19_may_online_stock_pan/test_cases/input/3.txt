[73,97,35,45,73,66,27,68,31,36,54,74,16,3,63,3,5,23,66,7]