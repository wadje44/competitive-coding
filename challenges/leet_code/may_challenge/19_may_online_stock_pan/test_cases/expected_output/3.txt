[1, 2, 1, 2, 3, 1, 1, 3, 1, 2, 3, 10, 1, 1, 3, 1, 2, 3, 7, 1]